src/dev/code_coverage/ingest_coverage/either.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/* eslint new-cap: 0 */
+/* eslint no-unused-vars: 0 */
+
+export const Right = x => ({
+  chain: f => f(x),
+  map: f => Right(f(x)),
+  fold: (f, g) => g(x),
+  inspect: () => `Right(${x})`,
+});
+
+Right.of = function of(x) {
```

Comment:
Is `Right.of` used?

Reply:
I think so, looking...

Reply:
the `.of`'s are just helpers that I use in other code I reckon.  I can drop them

Reply:
Oh yeah, they are used.  Sorry, I wrote this a while ago.